### ISSUE TITLE
`--cargo-args miri` does not work

### ISSUE THREAD

**Issue by matthiaskrgr** [Nilstrieb/cargo-minimize#5]
`cargo minimize --cargo-args "miri run" --verify-fn='|output| output.contains("dest.layout.ty.is_integral()")'`

````
 INFO cargo_minimize::processor Collecting file: src/main.rs
 INFO cargo_minimize::processor Initial build: no
ERROR cargo_minimize An error occured:
Initial build must reproduce issue. Output:
error: Found argument 'miri' which wasn't expected, or isn't valid in this context
````

**Comment by Nilstrieb** [comment]
It currently hard codes build and passes the cargo args additionally. It would make more sense if cargo args was defaulted to `build` instead.

**Comment by Nilstrieb** [comment]
Fixed in [c0289c6](https://github.com/Nilstrieb/cargo-minimize/commit/c0289c65f7011611cc675fca77b04560108e83bc), although `miri run` still won't work properly because it will run into everybody_loops.
